<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN"
    "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">

<html xml:lang="en" xmlns="http://www.w3.org/1999/xhtml" lang="en">
<head>
  <meta name="generator" content= />
  <meta http-equiv="imagetoolbar" content="no" />
  <meta content="http://www.unspam.com/noemailcollection" name=
  "no-email-collection" />

  <title>ffAMBER Homepage</title>
  <meta content="MSHTML 6.00.6000.16525" name="GENERATOR" />
  <meta content= "college, science, chemistry, biochemistry, department, university, csu, csulb, california, long beach, education, degrees, baccalaureate, graduate, alumni, seminars, classes, faculty, students, research, teaching, beach, los angeles, jobs, careers, employment, degrees in chemistry, degrees in biochemistry" name="keywords" />
  <meta content="index, follow" name="robots" />
  <meta content="Eric J Sorin" name="author" />
  <meta content="esorin@csulb.edu" name="owner" />
  <meta http-equiv="content-type" content= "text/html; charset=us-ascii" />
  <link href="style.css" type="text/css" rel="stylesheet" />
</head>

<body>
  <table class="topmast" summary="Header Layout" border="0"
  cellpadding="0" cellspacing="0" width="100%">
    <tbody>
      <tr>
        <td><a href="http://www.cnsm.csulb.edu/"><img alt=
        "College of Natural Sciences and Mathematics" src=
        "cnsm-blk.gif" border="0" /></a></td>

        <td align="right"><a href="http://www.csulb.edu/"><img alt=
        "California State University, Long Beach" src=
        "csulb-blk.gif" border="0" /></a></td>
      </tr>
    </tbody>
  </table>

  <!-- begin search form area -->
  <form action="http://www.google.com/u/lbsu" method="get" name=
  "search" class="sitesearchform" id="search">
    <table style="width: 100%;" class="search" summary=
    "Search Form" border="0" cellpadding="0" cellspacing="0">
      <tbody>
        <tr>
          <td>&nbsp;&nbsp;Department of <b>Chemistry and Biochemistry</b></td>
          <td align="right"><!-- ***** Begin search ***** -->
          <!-- Modify the following line to replace "collegepath"
        If your URL does not include www.csulb.edu then substitute your URL inurl:your <label class="grygreen" for="SiteSearch">Search</label> -->
          <input name="hq" value="inurl:chemistry.csulb.edu" type=
          "hidden" /><input id="SiteSearch" class="sitesearchbox"
          name="q" size="15" maxlength="120" tabindex="1" type=
          "text" /><!-- ***** End search ***** -->&nbsp;<br /></td>

          <td style="vertical-align: top; width: 27px;"><input src=
          "SEARCH_ICON_ROUND.gif" alt="Search" class=
          "sitesearchbutton" type="image" /></td>
        </tr>
      </tbody>
    </table>
  </form><!-- end search form area -->

  <table class="breadcrum" summary="Navigation Menu" border="0"
  cellpadding="0" cellspacing="0" width="100%">
    <tbody>
      <tr>
        <td>&nbsp;&nbsp;<a href=
        "http://chemistry.csulb.edu/">Home</a> &nbsp;|&nbsp;
        <a href=
        "http://chemistry.csulb.edu/index.php?module=pagemaster&amp;PAGE_user_op=view_page&amp;PAGE_id=91">
        About</a> &nbsp;|&nbsp; <a href=
        "http://chemistry.csulb.edu/index.php?module=pagemaster&amp;PAGE_user_op=view_page&amp;PAGE_id=88">
        People</a> &nbsp;|&nbsp; <a href=
        "http://my.csulb.edu/">myCSULB</a> &nbsp;|&nbsp; <a href=
        "http://beachboard.csulb.edu/">BeachBoard</a> &nbsp;|&nbsp;
        <a href="http://webmail.csulb.edu/">CSULB Webmail</a>
        &nbsp;|&nbsp; <a href=
        "http://chemistry.csulb.edu/index.php?module=pagemaster&amp;PAGE_user_op=view_page&amp;PAGE_id=61">
        Contact</a> &nbsp;|&nbsp; <a href=
        "http://chemistry.csulb.edu/index.php?module=pagemaster&amp;PAGE_user_op=view_page&amp;PAGE_id=142">
        Sitemap</a> &nbsp;|&nbsp; <a href=
        "http://chemistry.csulb.edu/index.php?module=pagemaster&amp;PAGE_user_op=view_page&amp;PAGE_id=145">
        Help</a> &nbsp;|&nbsp; <a href=
        "http://chemistry.csulb.edu/newsletter">Newsletter</a></td>
      </tr>
    </tbody>
  </table>

  <table summary="masthead layout" border="0" cellpadding="0"
  cellspacing="0" width="100%">
    <tbody>
      <tr class="masthead">
        <td><img alt="Department of Chemistry and Biochemistry"
        src="mast-2.07.jpg" border="0" /></td>
      </tr>
    </tbody>
  </table>

  <table class="breadcrum" summary="Breadcrums" border="0"
  cellpadding="0" cellspacing="0" width="100%">
    <tbody>
      <tr>
        <td>&nbsp;&nbsp;<a href="http://www.csulb.edu/">CSULB</a>
        &gt;&gt; <a href="http://www.cnsm.csulb.edu/">CNSM</a>
        &gt;&gt; Chemistry and Biochemistry</td>

        <td align="right">Last updated 13 July 2009 </td><!-- <td>
Your IP address is: </td> -->
      </tr>
    </tbody>
  </table>

  <table class="main" style="width: 705px; height: 4206px;" align=
  "center" cellpadding="0" cellspacing="0">
    <tbody>
      <tr>
        <td></td>

        <td colspan="1" class="content" style="text-align: center;"
        valign="top">
          <center>
            <b><big><big><big>ffAMBER</big></big></big></b><br />
            <br />
            <b><big>AMBER force field ports for the GROMACS
            molecular dynamics suite</big></b><br />
            <br />
            <big style="font-weight: bold;">Maintained by Eric J.
            Sorin, Ph.D.</big><br />
            <big>Assistant Professor, Department of Chemistry &amp;
            Biochemistry</big><br />
            <big>California State University, Long Beach<br />
            <br />
            With significant contributions from:<br style=
            "font-weight: bold;" />
            <span style="font-weight: bold;">&nbsp;Sanghyun Park Ph.D., </span> 
	    <span style="font-weight: bold;">Jory
            Z. Ruscio Ph.D., Erik Thompson*, and Kristin Haldeman*</span></big><br />
	    <BR>	
	    <small>*CSULB undergraduate students </small>
            <br />
            <hr />
            <br />

            <center>
              <table align="center" border="0" cellpadding="2">
                <tbody>
                  <tr>
                    <td>
                      <table align="center" border="0" cellpadding=
                      "2">
                        <tbody>
                          <tr>
                            <td align="center"><a href=
                            "http://ambermd.org/"><img style=
                            "border: 0px solid ; width: 100px; height: 100px;"
                            alt="" src="amber.jpg" /></a></td>
                          </tr>

                          <tr>
                            <td align="center"><a target="_blank"
                            href="http://amber.scripps.edu/">AMBER
                            homepage</a></td>
                          </tr>
                        </tbody>
                      </table>
                    </td>

                    <td>
                      <table align="center" border="0" cellpadding=
                      "2">
                        <tbody>
                          <tr>
                            <td align="center">
                            <b><big>CONTENTS:</big></b></td>
                          </tr>

                          <tr>
                            <td align="center"><b><a href=
                            "#general">General
                            Information</a></b></td>
                          </tr>

                          <tr>
                            <td align="center"><b><a href=
                            "#ffinfo">Force Field
                            Information</a></b></td>
                          </tr>

                          <tr>
                            <td align="center"><b><a href=
                            "#validation">Implementation &amp;
                            Validation</a></b></td>
                          </tr>

                          <tr>
                            <td align="center"><b><a href=
                            "#install">Installation &amp;
                            Testing</a></b></td>
                          </tr>

                          <tr>
                            <td align="center"><b><font color=
                            "red">!</font> <a href=
                            "#usage">Important notes on using the
                            AMBER ports</a><font color=
                            "red">!</font></b></td>
                          </tr>

                          <tr>
                            <td align="center"><b><a href=
                            "#contact">Contact Us</a></b></td>
                          </tr>

                          <tr>
                            <td align="center"><b><a href=
                            "FAQ.html">ffAMBER FAQ</a></b></td>
                          </tr>

                          <tr>
                            <td align="center"><b><a href=
                            "tools.html">ffAMBER Tools</a></b></td>
                          </tr>
                        </tbody>
                      </table>
                    </td>

                    <td>
                      <table align="center" border="0" cellpadding=
                      "2">
                        <tbody>
                          <tr>
                            <td align="center"><a href=
                            "http://www.gromacs.org/"><img style=
                            "border: 0px solid ; width: 100px; height: 100px;"
                            alt="" src="gromacs.jpg" /></a></td>
                          </tr>

                          <tr>
                            <td align="center"><a target="_blank"
                            href="http://www.gromacs.org/">GROMACS
                            homepage</a></td>
                          </tr>
                        </tbody>
                      </table>
                    </td>
                  </tr>
                </tbody>
              </table><br />

              <table align="center" border="1" cellpadding="2">
                <tbody>
                  <tr>
                    <td align="center">Are you using our
                    <b>ffamber</b> ports for an upcoming
                    publication? <small><br /></small> If so,
                    please cite: <small><br /></small> Sorin &amp;
                    Pande (<b>2005</b>). <i>Biophys. J.</i>
                    <strong>88</strong>(4), 2472-2493.</td>
                  </tr>
                </tbody>
              </table><br />
              <hr />
              <br />
            </center>

            <p align="left"><b><big><a name="news" id=
            "general"><font color=red>Updates:</font></a></b>
	    We are currently working on integrating both the new ion parameters from
	    the Cheatham Group as well as the 2007 BSC0 version of the AMBER-99 force 
	    field for nucleic acids, both of which will be available this summer '09. 
	    </big></p>	
            <hr />

            <p align="left"><b><big><a name="general" id=
            "general">General Information:</a></big></b></p>

            <div align="justify">
              We have ported the following AMBER potentials and TIP
              water models for use in the GROMACS MD suite. AMBER
              ports for GROMACS versions 3.1.4, 3.2.1, and
              3.3/3.3.1 have been tested against AMBER 8.0, as
              discussed below. As there have been several versions
              of the TIP parameters published, we have taken them
              from the most recent paper of the Jorgensen group to
              describe TIP water (Mahoney, 2000). In addition to
              these force fields, we have added several ions, the
              urea molecule (URE), hydroxyproline (HYP), ornithine
              (ORN), and diaminobutyric acid (DAB), as described
              below. Please cite the following tabulated citations
              when using any of these force fields.<br />
              <br />
            </div>

            <div align="justify">
              AMBER conventions have been maintained where
              possible. However, due to specific GROMACS
              functionality this was not always feasible. As
              detailed in this page, there are several very
              important caveats to using these AMBER ports properly
              within the GROMACS suite. <font color="red"><b>Please
              read the following information carefully before using
              these AMBER ports</b></font>, and refer also to the
              <a href=
              "http://chemistry.csulb.edu/ffamber/FAQ.html">FAQ</a>
              if you have questions that are not answered below.
              This index.html and supporting files can be found in
              the README subdirectory of the ffamber*tar.gz
              downloads, which are available with or without the
              pdf documentation listed in the table below.<br />
              <br />
            </div>

            <div align="justify">
              Due to the caveats mentioned above, ffamber ports are
              not yet distributed as part of the GROMACS package.
              We are currently working with GROMACS developers Erik
              Lindahl &amp; David van der Spoel to make the
              necessary changes in GROMACS that will allow us to
              fully merge these ports with upcoming GROMACS
              distributions and thereby simplify the use of AMBER
              in GROMACS.
            </div><br />

            <center>
              <div align="center">
                <table style="width: 686px; height: 257px;" border=
                "1" cellpadding="1" cellspacing="1">
                  <tbody>
                    <tr>
                      <td align="center"><b>Potential</b></td>

                      <td align="center"><b>Port Name</b></td>

                      <td align="center"><b>Literature</b></td>

                      <td align="center">DL</td>
                    </tr>

                    <tr>
                      <td style="text-align: left;">AMBER-94</td>

                      <td style="text-align: left;">ffamber94</td>

                      <td style="text-align: left;">Cornell et al.
                      (<b>1995</b>), <i>JACS</i> <b>117</b>,
                      5179-5197</td>

                      <td><a target="_blank" href=
                      "pdfs/cornell_amber94_1995jacs.pdf">PDF</a></td>
                    </tr>

                    <tr>
                      <td style="text-align: left;">AMBER-96</td>

                      <td style="text-align: left;">ffamber96</td>

                      <td style="text-align: left;">Kollman
                      (<b>1996</b>), <i>Acc. Chem. Res.</i>
                      <b>29</b>, 461-469</td>

                      <td><a target="_blank" href=
                      "pdfs/kollman_amber_1996acr.pdf">PDF</a></td>
                    </tr>

                    <tr>
                      <td style="text-align: left;">AMBER-99</td>

                      <td style="text-align: left;">ffamber99</td>

                      <td style="text-align: left;">Wang et al.
                      (<b>2000</b>), <i>J. Comp. Chem.</i>
                      <b>21</b>, 1049-1074</td>

                      <td><a target="_blank" href=
                      "pdfs/wang_amber99_2000jcc.pdf">PDF</a></td>
                    </tr>

                    <tr>
                      <td style="text-align: left;">AMBER-03*</td>

                      <td style="text-align: left;">ffamber03</td>

                      <td style="text-align: left;">Duan et al.
                      (<b>2003</b>), <i>J. Comp. Chem.</i>
                      <b>24</b>, 1999-2012</td>

                      <td><a target="_blank" href=
                      "pdfs/duan_AMBER03_2003jcc.pdf">PDF</a></td>
                    </tr>

                    <tr>
                      <td style="text-align: left;">AMBER-GS</td>

                      <td style="text-align: left;">ffamberGS</td>

                      <td style="text-align: left;">Garcia &amp;
                      Sanbonmatsu (<b>2002</b>), <i>PNAS</i>
                      <b>99</b>, 2782-2787</td>

                      <td><a target="_blank" href=
                      "pdfs/garcia_amberGS_2002pnas.pdf">PDF</a></td>
                    </tr>

                    <tr>
                      <td style="text-align: left;">
                      AMBER-GS-S**</td>

                      <td style="text-align: left;">ffamberGSs</td>

                      <td style="text-align: left;">Nymeyer &amp;
                      Garcia (<b>2003</b>) <i>PNAS</i> <b>100</b>,
                      13934-13939</td>

                      <td><a target="_blank" href=
                      "pdfs/garcia_amberGS-S_2003pnas.pdf">PDF</a></td>
                    </tr>

                    <tr>
                      <td style="text-align: left;">
                      AMBER-99<font face="symbol">f</font>***</td>

                      <td style="text-align: left;">ffamber99p</td>

                      <td style="text-align: left;">Sorin &amp;
                      Pande (<b>2005</b>). <i>Biophys. J.</i>
                      <strong>88</strong>(4), 2472-2493</td>

                      <td><a target="_blank" href=
                      "pdfs/sorin_amber99phi_2005bj.pdf">PDF</a></td>
                    </tr>

                    <tr>
                      <td style="text-align: left;">AMBER-99SB</td>

                      <td style="text-align: left;">
                      ffamber99sb</td>

                      <td style="text-align: left;">Hornak et. al
                      (<b>2006</b>). <i>Proteins</i>
                      <strong>65</strong>, 712-725</td>

                      <td><a target="_blank" href=
                      "pdfs/simmerling_amber99sb_2006proteins.pdf">PDF</a></td>
                    </tr>

                    <tr>
                      <td style="text-align: left;">TIP3P</td>

                      <td style="text-align: left;">HOH or T3P</td>

                      <td style="text-align: left;">Mahoney &amp;
                      Jorgensen (<b>2000</b>). <i>J. Chem.
                      Phys.</i> <strong>112</strong>,
                      8910-8922</td>

                      <td><a target="_blank" href=
                      "pdfs/mahoney_tip5p_2000jcp.pdf">PDF</a></td>
                    </tr>

                    <tr>
                      <td style="text-align: left;">
                      TIP3P(heavy)****</td>

                      <td style="text-align: left;">T3H</td>

                      <td style="text-align: left;">Mahoney &amp;
                      Jorgensen (<b>2000</b>). <i>J. Chem.
                      Phys.</i> <strong>112</strong>,
                      8910-8922</td>

                      <td><a target="_blank" href=
                      "pdfs/mahoney_tip5p_2000jcp.pdf">PDF</a></td>
                    </tr>

                    <tr>
                      <td style="text-align: left;">TIP4P</td>

                      <td style="text-align: left;">T4P</td>

                      <td style="text-align: left;">Mahoney &amp;
                      Jorgensen (<b>2000</b>). <i>J. Chem.
                      Phys.</i> <strong>112</strong>,
                      8910-8922</td>

                      <td><a target="_blank" href=
                      "pdfs/mahoney_tip5p_2000jcp.pdf">PDF</a></td>
                    </tr>

                    <tr>
                      <td style="text-align: left;">TIP4P-Ew</td>

                      <td style="text-align: left;">T4E</td>

                      <td style="text-align: left;">Horn et al.
                      (<b>2004</b>). <i>J. Chem.
                      Phys.</i><strong>120</strong>, 9665-9678</td>

                      <td><a target="_blank" href=
                      "pdfs/horn_tip4pEW_2004jcp.pdf">PDF</a></td>
                    </tr>

                    <tr>
                      <td style="text-align: left;">TIP5P</td>

                      <td style="text-align: left;">T5P</td>

                      <td style="text-align: left;">Mahoney &amp;
                      Jorgensen (<b>2000</b>). <i>J. Chem.
                      Phys.</i> <strong>112</strong>,
                      8910-8922</td>

                      <td><a target="_blank" href=
                      "pdfs/mahoney_tip5p_2000jcp.pdf">PDF</a></td>
                    </tr>

                    <tr>
                      <td></td>
                    </tr>
                  </tbody>
                </table>
              </div>
            </center>

            <div style="margin-left: 76px; width: 750px;" align=
            "left">
              * all-atom only, does NOT include the AMBER-03ua
              united-atom potential<br />
              ** no 1-4 vdW scaling<br />
              *** helix-coil force field<br />
              **** GROMACS heavy H2O definition using TIP3P
              parameters
            </div>
          </center><br />
          <hr />

          <p align="left"><b><big><a name="ffinfo" id=
          "ffinfo">Force Field Information:</a></big></b></p>

          <div align="justify">
            <p><b><font color="red">AMBER-94</font></b> is one of
            the most widely used and understood of the seminal
            all-atom force fields, though known to be somewhat
            "helix-friendly."</p>

            <p><b><font color="red">AMBER-96</font></b> is a
            variant of AMBER-94 in which peptide <font face=
            "symbol">f/y</font> torsional potentials have been
            adjusted to yeild better agreement between molecular
            mechanical and quantum mechanical energetics
            (disfavoring helices, favoring extended
            conformations).</p>

            <p><b><font color="red">AMBER-GS</font></b> is
            identical to AMBER-94 with peptide <font face=
            "symbol">f/y</font> torsional potentials removed (set
            to zero). This AMBER-GS differs from Angel Garcia's
            force field with a
            minor modification: Garcia's version also does *NOT*
            scale 1-4 vdW interactions (personal comm., A. Garcia),
            as is standard for all AMBER force fields. Please note
            this distinction when choosing between these force
            fields [for more information, see Sorin &amp; Pande
            (<b>2005</b>). <i>J. Comp. Chem.</i> <b>26</b> (7),
            682-690].  To use the "true AMBER-GS" force field that 
	    removes 1-4 vdW scaling, which we have referred to as 
	    <b><font color="red">AMBER-GS-S</font></b>,
	    please use the ffamberGSs files provided in our ffAMBER ports.</p>

            <p><b><font color="red">AMBER-99</font></b> is the "3rd
            generation" update to AMBER-94, including updated
            parameters for both amino and nucleic acids. The
            primary changes for peptides/proteins is again in the
            <font face="symbol">f/y</font> torsional potentials
            (again to disfavor helical conformations).</p>

            <p><b><font color="red">AMBER-99<font face=
            "symbol">f</font></font></b> is our AMBER-99 variant
            for helix-coil simulations and replaces the <font face=
            "symbol">f</font> torsion in AMBER-99 with that of the
            Cornell (AMBER-94) potential.</p>

            <p><b><font color="red">AMBER-99SB</font></b> is the
            AMBER-99 variant developed by Simmerling and co-workers
            with updated backbone <font face="symbol">f</font> and
            <font face="symbol">y</font> torsions fit to ab initio
            calculations (ported by Jory Z. Ruscio).</p>

            <p><b><font color="red">AMBER-03</font></b> is a
            variant of the AMBER-99 potential in which charges and
            main-chain torsion potentials have been rederived based
            on QM+continuum solvent calculations and each amino
            acid is allowed unique main-chain charges (nucleic
            acids have not been modified from AMBER-99).</p>

            <p>For more information on each of these FF's, please
            refer to the <a target="_blank" href=
            "http://amber.scripps.edu/">AMBER homepage</a> and the
            references above.<br />
            <br />
            <br />
            <b>The following additions have also been
            made:</b><br />
            <br />
            Ornithine (ORN) and diaminobutyric acid (DAB) residues
            were defined using standard AMBER-94 and AMBER-99
            Lennard-Jones and bonded parameters. ORN sidechain
            charges were taken from the AMBER-99 port in <a target=
            "_blank" href="http://dasher.wustl.edu/tinker/">TINKER
            4.0</a>; DAB sidechain charges were fit to maintain
            proper LYS --&gt; ORN --&gt; DAB heavy atom charge
            group trends, as described in:<br />
            B. Zagrovic, J. Lipfert, E.J. Sorin, I.S. Millett, W.F.
            Van Gunsteren, S. Doniach, &amp; V.S. Pande
            (<b>2005</b>) <i>PNAS</i>, <b>102</b>(33),
            11698-11703.<br />
            <br />
            The hydroxyproline (HYP) residue has also been added*
            in accord to the following publications:<br />
            S.D. Mooney, P.A. Kollman &amp; T.E. Klein
            (<b>2002</b>), <i>Biopolymers</i> <b>64</b>,
            63-71.<br />
            S. Park, R.J. Radmer, T.E. Klein &amp; V.S. Pande
            (<b>2005</b>), <i>J. Comp. Chem.</i> <b>26</b>,
            1612&ndash;1616.<br />
            *due to the significant force field differences, HYP is
            not present in AMBER-03 ports<br />
            <br />
            The urea molecule (URE) was defined using standard
            AMBER Lennard-Jones and bonded parameters, and RESP
            charges were derived by Jim Caldwell.</p>
          </div>

          <p align="left"><br /></p>
          <hr />
          <br />

          <div style="text-align: left;">
            <b><big><a name="validation" id=
            "validation">Implementation &amp;
            Validation:</a></big></b>
          </div>

          <div align="justify">
            <br />
            The AMBER-94 and AMBER-99 force fields were ported and
            validated first. Force field files for the variants of
            AMBER-94 and AMBER-99 were then generated after
            validation by making the minor modifications required
            (i.e. peptide <font face="symbol">f/y</font> and 1-4
            vdW scaling).<br />
            <br />
            While newer GROMACS distributions support multiple
            torsional terms for a given atomic quartet, v3.1.4 did
            not. Proper torsion potentials were therefore converted
            to Ryckaert-Bellemans series in a fashion similar to
            that described in the GROMACS manual for OPLS torsions.
            Additionally, improper torsions are handled in GROMACS
            using the proper dihedral definition to match the AMBER
            standard, rather than the default analytical function
            used for improper dihedrals in GROMACS. Because we
            found inconsistencies in the definitions of improper
            torsions when using the AMBER 8.0 package (i.e.
            ordering of the atomic quartet), impropers in our ports
            were set to agree with AMBER 8.0, and proper torsions
            were then checked for exact agreement between all AMBER
            8.0 Fourier series torsions and the GROMACS
            Ryckaert-Bellemans series torsions.<br />
            <br />
            To validate the potential energetics of our ports,
            structures for all monomer, homodimer, and homotrimer
            systems (including amino and nucleic acids) were
            generated using the LEAP program in AMBER and these
            structures were perturbed by adding a random number
            between -0.2 and +0.2 Angstroms to all 3N coordinates.
            This step is necessary because default structures
            generated by AMBER's LEAP program contain many low
            energy terms which are not suitable for quantitative
            force field validation. All potential energy components
            were compared between GROMACS 3.1.4/3.2.1/3.3/3.3.1 and
            AMBER 8.0 and the resulting energy differences were
            generally &lt; 0.005%. These results were not
            significantly altered by taking the mean over 10
            independent perturbations for each monomer/dimer/trimer
            system or altering the magnitude of the applied 3N
            perturbations.
          </div><br />

          <center>
            <big><b>Mean relative errors for all<br />
            monomers, homodimers, and homotrimers*<br /></b></big>
            (using single precision binaries)<br />

            <table border="1" cellpadding="1" cellspacing="1"
            width="50%">
              <tbody>
                <tr>
                  <td rowspan="2" align="center"><b>&lt;|Error|&gt;
                  (%)</b></td>

                  <td colspan="2" align="center">AMBER-94</td>

                  <td colspan="2" align="center">AMBER-99</td>

                  <td colspan="2" align="center">AMBER-03</td>
                </tr>

                <tr>
                  <td align="center">Amino</td>

                  <td align="center">Nucleic</td>

                  <td align="center">Amino</td>

                  <td align="center">Nucleic</td>

                  <td align="center">Amino</td>

                  <td align="center">Nucleic<sup>**</sup></td>
                </tr>

                <tr>
                  <td align="center">BOND</td>

                  <td align="center">0.000</td>

                  <td align="center">0.000</td>

                  <td align="center">0.000</td>

                  <td align="center">0.000</td>

                  <td align="center">0.000</td>

                  <td align="center">0.000</td>
                </tr>

                <tr>
                  <td align="center">ANGLE</td>

                  <td align="center">0.000</td>

                  <td align="center">0.000</td>

                  <td align="center">0.000</td>

                  <td align="center">0.000</td>

                  <td align="center">0.000</td>

                  <td align="center">0.000</td>
                </tr>

                <tr>
                  <td align="center">DIHEDRAL</td>

                  <td align="center">0.004</td>

                  <td align="center">0.000</td>

                  <td align="center">0.003</td>

                  <td align="center">0.000</td>

                  <td align="center">0.005</td>

                  <td align="center">0.000</td>
                </tr>

                <tr>
                  <td align="center">1-4 vdW</td>

                  <td align="center">0.000</td>

                  <td align="center">0.001</td>

                  <td align="center">0.001</td>

                  <td align="center">0.001</td>

                  <td align="center">0.000</td>

                  <td align="center">0.001</td>
                </tr>

                <tr>
                  <td align="center">1-4 QQ</td>

                  <td align="center">0.001</td>

                  <td align="center">0.001</td>

                  <td align="center">0.001</td>

                  <td align="center">0.001</td>

                  <td align="center">0.001</td>

                  <td align="center">0.001</td>
                </tr>

                <tr>
                  <td align="center">vdW</td>

                  <td align="center">0.001</td>

                  <td align="center">0.002</td>

                  <td align="center">0.001</td>

                  <td align="center">0.006</td>

                  <td align="center">0.001</td>

                  <td align="center">0.006</td>
                </tr>

                <tr>
                  <td align="center">QQ</td>

                  <td align="center">0.003</td>

                  <td align="center">0.004</td>

                  <td align="center">0.003</td>

                  <td align="center">0.003</td>

                  <td align="center">0.003</td>

                  <td align="center">0.003</td>
                </tr>

                <tr>
                  <td align="center">Total Potential</td>

                  <td align="center">0.001</td>

                  <td align="center">0.001</td>

                  <td align="center">0.001</td>

                  <td align="center">0.001</td>

                  <td align="center">0.001</td>

                  <td align="center">0.001</td>
                </tr>

                <tr>
                  <td></td>
                </tr>

                <tr>
                  <td></td>
                </tr>
              </tbody>
            </table>*highest mean error from GROMACS 3.1.4, 3.2.1,
            3.3/3.3.1<br />
            **unchanged from AMBER-99, tested for consistency
          </center>

          <p align="left"><br /></p>
          <hr />
          <br />

          <div style="text-align: left;">
            <b><big><a name="install" id="install">Installation
            &amp; Testing:</a></big></b>
          </div>

          <table border="0" cellpadding="2" cellspacing="1" width=
          "100%">
            <tbody>
              <tr>
                <td valign="top">(1)</td>

                <td style="text-align: left;">Install the desired
                <a target="_blank" href=
                "http://www.gromacs.org/">GROMACS</a> distribution
                (v3.1.4, v3.2.1, v3.3, or v3.3.1).</td>
              </tr>

              <tr>
                <td valign="top">(2)</td>

                <td style="text-align: left;">Download the
                appropriate ffamber ports (.tar.gz) with or without
                pdf documentation from the table below, being sure
                that the version number you choose matches the
                version of GROMACS you are using.</td>
              </tr>

              <tr>
                <td valign="top">(3)</td>

                <td style="text-align: left;">Unzip/untar the
                downloaded tar.gz file.</td>
              </tr>

              <tr>
                <td valign="top">(4)</td>

                <td style="text-align: left;">Copy aminoacids.dat
                and vdwradii.dat to the "top" directory in your
                gromacs distribution (you should see force field
                files there, such as ffoplsaa.*). If you plan on
                simulating nucleic acids, refer to the <a href=
                "#aadat">note for nucleic acids in
                aminoacids.dat</a> below.</td>
              </tr>

              <tr>
                <td valign="top">(5)</td>

                <td style="text-align: left;">Files for each force
                field are located in a seperate subdirectory, such
                as ffamber94/ for the Cornell potential. Copy the
                desired ffamber* files to the top directory in your
                gromacs distribution.</td>
              </tr>

              <tr>
                <td valign="top">(6)</td>

                <td style="text-align: left;">Increment the number
                at the top of the "top/FF.dat" file by 1 for each
                AMBER port you'll install (so that it matches the
                total number of forcefields available in the "top"
                directory).</td>
              </tr>

              <tr>
                <td valign="top">(7)</td>

                <td style="text-align: left;">Add lines like the
                following to the "top/FF.dat" file. These are used
                by pdb2gmx to allow you to identify the desired FF
                and field 1 must match the ffamber* filename
                prefixes, whereas the following fields can be
                user-defined:<br />
                &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;ffamber94 AMBER94
                Cornell protein/nucleic forcefield<br />
                &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;ffamber99 AMBER99
                Wang protein/nucleic acid forcefield<br />
                &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;ffamber99p AMBER99p
                protein/nucleic forcefield<br />
                &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;ffamber03 AMBER03
                Duan protein/nucleic forcefield</td>
              </tr>

              <tr>
                <td valign="top">(8)</td>

                <td style="text-align: left;">Locate the GMXRC in
                your GROMACS distribution and run `source
                GMXRC`.</td>
              </tr>

              <tr>
                <td valign="top">(9)</td>

                <td style="text-align: left;">
                  Run `pdb2gmx -H14 -f any.pdb` with any pdb to
                  verify that these force fields are now seen by
                  GROMACS. Working example .pdb files are available
                  below, alongside pre-prepared gro and top files (GROMACS 3.1.4
		   / AMBER94) to which you can compare your 
		  resulting files.<BR><BR>

                  <center>
		  <TABLE cellpadding="8">
		  <TR>
                    <TD>Protein</TD>
		    <TD>DNA</TD>
 		    <TD>RNA</TD>
		  <TR>
                    <TD><a href="villin.pdb">Villin headpeice</a></TD>
		    <TD><a href="dickerson.pdb">Dickerson dodecamer</a></TD>
 		    <TD><a href="gcaa.pdb">GCAA tetraloop hairpin</a></TD>
                  </center>
		  <TR>
                  <center>
                    <TD><a href="villin.gro">AMBER94 gro file</a></TD>
		    <TD><a href="dickerson.gro">AMBER94 gro file</a></TD>
		    <TD><a href="gcaa.gro">AMBER94 gro file</a> </TD>
                  </center>
	          <TR>
                  <center>
                    <TD><a href="villin.top">AMBER94 top file</a> </TD>
		    <TD><a href="dickerson.top">AMBER94 top file</a></TD>
		    <TD><a href="gcaa.top">AMBER94 top file</a></TD>
		    <TR>
		  </TABLE>
                  </center>
		  <BR>
                </td>
              </tr>

              <tr>
                <td valign="top">(10)</td>

                <td style="text-align: left;">Modify residue names
                in your pdb file(s) as described below to properly
                generate .gro and .top files using pdb2gmx (the
                files above are ready to use).</td>
              </tr>

              <tr>
                <td></td>
              </tr>

              <tr>
                <td></td>
              </tr>
            </tbody>
          </table><br />

          <center>
            <b><big>AMBER port distributions</big></b>

            <table style="width: 722px; height: 159px;" border="1"
            cellpadding="1" cellspacing="1">
              <tbody>
                <tr>
                  <td align="center"><b>Version</b></td>

                  <td align="center"><b>Original<br />
                  Release Date</b></td>

                  <td align="center"><b>Download</b></td>

                  <td align="center"><b>Notes on this
                  release</b></td>
                </tr>

                <tr>
                  <td align="center">3.1.4</td>

                  <td align="center">July 2005</td>

                  <td align="center"><a href=
                  "ffamber_v3.1.4-doc.tar.gz">with pdfs</a><br />
                  <a href="ffamber_v3.1.4.tar.gz">without
                  pdfs</a></td>

                  <td align="center">Must use -H14 flag in
                  pdb2gmx</td>
                </tr>

                <tr>
                  <td align="center">3.2.1</td>

                  <td align="center">July 2005</td>

                  <td align="center"><a href=
                  "ffamber_v3.2.1-doc.tar.gz">with pdfs</a><br />
                  <a href="ffamber_v3.2.1.tar.gz">without
                  pdfs</a></td>

                  <td align="center">-H14 flag no longer necessary
                  for pdb2gmx</td>
                </tr>

                <tr>
                  <td align="center">3.3</td>

                  <td align="center">Nov. 2005</td>

                  <td align="center"><a href=
                  "ffamber_v3.3-doc.tar.gz">with pdfs</a><br />
                  <a href="ffamber_v3.3.tar.gz">without
                  pdfs</a></td>

                  <td align="center">.hdb file format change,<br />
                  hydrogen database now functional for nucleic
                  acids</td>
                </tr>

                <tr>
                  <td align="center">3.3.1</td>

                  <td align="center">May 2006</td>

                  <td align="center"><a href=
                  "ffamber_v3.3.1-doc.tar.gz">with pdfs</a><br />
                  <a href="ffamber_v3.3.1.tar.gz">without
                  pdfs</a></td>

                  <td align="center">This port has also been validated using GMX 3.3.3, 
		 <br />but may give warning or error messages using GMX 3.3.2</td>
                </tr>

                <tr>
                  <td align="center">4.0</td>

                  <td align="center">Feb 2009</td>

                  <td align="center"><a href=
                  "ffamber_v4.0-doc.tar.gz">with pdfs</a><br />
                  <a href="ffamber_v4.0.tar.gz">without
                  pdfs</a></td>

                  <td align="center">This port has been validated using GMX 4.0.2, 4.0.3, & 4.0.4 
		 <br />NOTE: pdb2gmx auto-converts residue CYM to CYN in 4.0.2 & 4.0.3</td>
                </tr>


                <tr>
                  <td></td>
                </tr>
              </tbody>
            </table>
          </center>

          <p style="text-align: center; font-weight: bold;">
	  ffAMBER tar.gz files are ~7.4 MB with .pdfs and ~650 kB without</p>	

          <p style="text-align: center; font-weight: bold;"></p>
          <hr />
          <br />

          <div style="text-align: left;">
            <b><big><a name="usage" id="usage">Important notes on
            using the AMBER ports:</a></big></b>
          </div>

          <div align="justify">
            <br />
            This list includes only the most important information
            needed to properly use our AMBER ports inside the
            GROMACS suite. Non-vital information is compiled in the
            <a href="FAQ.html">FAQ</a> listed above. Information on
            using GROMACS as well as FAQs, tutorials, and a user
            forum are available at <a target="_blank" href=
            "http://www.gromacs.org/">http://www.gromacs.org/</a>.
          </div><br />

          <table border="0" cellpadding="2" cellspacing="1" width=
          "100%">
            <tbody>
              <tr>
                <td valign="top">(1)</td>

                <td>
                  <div align="justify">
                    <b><font color="red">(!) -H14 flag in
                    gromacs:</font></b> You must use the -H14 flag
                    when running the GROMACS v3.1.4 version of
                    pdb2gmx to get *ALL* hydrogen-hydrogen 1-4
                    interactions that should be present in AMBER
                    topologies. This has been corrected in v3.2.1
                    and the -H14 flag is no longer necessary.
                  </div>
                </td>
              </tr>

              <tr>
                <td valign="top">(2)</td>

                <td>
                  <div align="justify">
                    <b><font color="red">(!) Residue
                    Nomenclature:</font></b> Residues in the AMBER
                    ports are named according to their position in
                    the sequence (i.e. terminal, non-terminal,
                    monomer) following standard AMBER conventions.
                    For this reason, it may be necessary to rename
                    residues in the .pdb file you will import
                    beforehand. Please note that all residues are
                    named in the residue topology files (i.e.
                    ffamber*.rtp), so if you are unsure of the
                    correct residue name to use, you should be able
                    to find it there. The .rtp files are ordered as
                    follows: water models (TIP), ions &amp; urea
                    (URE), peptide terminal capping residues (ie.
                    ACE, NH2, NMe), non-terminal amino acids (i.e.
                    TYR, ALA), non-terminal acidic amino acids
                    (i.e. ASH, GLH, etc.), C-terminal amino acids
                    (i.e. CALA, CGLY), N-terminal amino acids (i.e.
                    NALA, NGLY), and all nucleic acid residues.
                    Nucleic acids listed at the end of each .rtp
                    follow the following order for each residue
                    type: DNA is first, followed by RNA, in the
                    order 5'-term, 3'-term, non-terminal, and
                    monomer. The three .pdb files above are
                    examples of how pdb files shoud be modified.
                    Residues in the ffamber ports have been named
                    as follows:

                    <table>
                      <tbody>
                        <tr>
                          <td valign="top"><b><font color=
                          "red">(a)</font></b></td>

                          <td>
                            <div align="justify">
                              <b>Non-terminal amino and nucleic
                              acid residues</b> follow standard
                              AMBER naming conventions. To avoid
                              confusion between GROMACS and AMBER
                              conventions, we have omitted the
                              redundant HIS residue, leaving HID,
                              HIE, HIP, and terminal versions of
                              these topologies. Additionally, due
                              to the automated changing of certain
                              residue names by pdb2gmx, the LYS and
                              CYS residues have been renamed LYP
                              (Lysine plus) and CYN (Cysteine
                              neutral, compared to AMBER residue
                              CYM = Cysteine minus).
                            </div>
                          </td>
                        </tr>

                        <tr>
                          <td valign="top"><b><font color=
                          "red">(b)</font></b></td>

                          <td>
                            <div align="justify">
                              <b>C- and N-terminal amino acids</b>
                              include a C or N prefix respectively,
                              so C-terminal ALA is CALA and
                              N-ternimal PHE is NPHE. As with
                              non-terminal versions, the LYS and
                              CYS terminal residues are listed as
                              NLYP,CLYP and NCYN,CCYN.
                            </div>
                          </td>
                        </tr>

                        <tr>
                          <td valign="top"><b><font color=
                          "red">(c)</font></b></td>

                          <td>
                            <div align="justify">
                              <b>Nucleic acid residues</b> come in
                              four flavors. All residue names
                              include XY, where X = D or R for DNA
                              or RNA respectively, and Y = first
                              letter of the nucleotide name. A
                              suffix (XY<u><b>Z</b></u>) is added
                              for monomers (Z=N), 5'-terminal
                              (Z=5), and 3'-terminal (Z=3)
                              residues. For example, 3'-term DNA
                              Cytosine = "DC3", 5'-term RNA
                              Cytosine = "RC5", non-terminal DNA
                              Cytosine = "DC", and lone RNA
                              Cytosine monomer = "RCN".
                            </div>
                          </td>
                        </tr>

                        <tr>
                          <td valign="top"><b><font color=
                          "red">(d)</font></b></td>

                          <td>
                            <div align="justify">
                              <b>Cys disulfide bonds</b> can be
                              instituted by changing the names of
                              the CYS residues involved in the
                              disulfide bonds to CYS2.
                            </div>
                          </td>
                        </tr>

                        <tr>
                          <td></td>
                        </tr>
                      </tbody>
                    </table>
                  </div>
                </td>
              </tr>

              <tr>
                <td valign="top">(3)</td>

                <td>
                  <div align="justify">
                    <b><font color="red">(!) Atom
                    Nomenclature:</font></b> Once you have modified
                    the residue names in your pdb file according to
                    the rules above, pdb2gmx may report a fatal
                    error like "Fatal error: Atom AA in residue XYZ
                    not found in rtp entry with NN atoms while
                    sorting atoms." This may or may not be caused
                    by the use of an aminoacids.dat file in the
                    "top" directory of your GROMACS distribution
                    that does not include the name of the residue
                    that is causing the problem (XYZ in this case).
                    This can be easily fixed by modifying the name
                    of the problematic atom in your .pdb file to
                    match what is shown for residue XYZ in the .rtp
                    file.
                  </div>
                </td>
              </tr>

              <tr>
                <td valign="top">(4)</td>

                <td>
                  <div align="justify">
                    <b><font color="red">(!) Water
                    nomenclature:</font></b> residues in
                    pre-solvated pdb files should be named as shown
                    here to use the correct water model in the
                    desired amber port: TIP3P ("HOH" or "T3P"),
                    heavy TIP3P ("T3H"), TIP4P ("T4P"), TIP4P-Ew
                    ("T4E"), and TIP5P ("T5P"). This usage allows
                    any model to be used without modifying the
                    force field (.rtp) files. Note, however, that
                    for the default "HOH" listing the TIP3P model
                    will be is assumed. Also note that because
                    GROMACS supports SOL=HOH water definitions,
                    importing a solvated PDB with names other than
                    HOH or SOL may cause the solvent molecules to
                    be treated as part of the biopolymer rather
                    than listing SOL molecules at the end of the
                    topology, which has to be modified by hand
                    after the fact.
                  </div>
                </td>
              </tr>

              <tr>
                <td valign="top">(5)</td>

                <td>
                  <div align="justify">
                    <b><font color="red">(!) Specifying water
                    models in .top files:</font></b> If not using
                    the default SPC water model in GROMACS, make
                    sure you list the correct ffamber_tip*.itp file
                    in your gromacs topology file! TIP3P heavy
                    water (H's four times heavier, difference
                    subtracted from oxygen) is defined in
                    ffamber_tip3p_heavy.itp, and the TIP4P-Ewald
                    model uses ffamber_tip4pEW.itp. All other
                    models are specified in the ffamber_tipXp.itp
                    files, where X = {3,4,5}. Water boxes are also
                    present, with the .gro file extension.
                  </div>
                </td>
              </tr>

              <tr>
                <td valign="top">(6)</td>

                <td>
                  <div align="justify">
                    <b><a name="aadat" id="aadat"><font color=
                    "red">(!) For nucleic acid
                    simulations:</font></a></b> Amino acid residues
                    have been added to aminoacids.dat, which (i)
                    allows easy use of GROMACS analysis tools and
                    (ii) aids GROMACS in matching atoms in a pdb
                    with those in the residue topology (.rtp) file.
                    However, for nucleic acids this also often
                    causes pdb2gmx to replace an H atom in the
                    first residue of all nucleic acid chains with
                    an incorrect H atom, resulting in non-neutral
                    charge. The correct atom is generally replaced
                    with an atom of type amberXX_25 (hydroxyl H),
                    as pdb2gmx treats it as a terminal hydrogen.
                    For this reason, we provide both the standard
                    <a href=
                    "ffamber_v3.1.4/aminoacids.dat">aminoacids.dat</a>
                    (without nucleic acids) and a second version
                    with nucleic acid residues included (<a href=
                    "ffamber_v3.1.4/aminoacids-NA.dat">aminoacids-NA.dat</a>).
                    You can run <a href="gcaa.pdb">gcaa.pdb</a>
                    with both versions of aminoacids.dat by copying
                    either file into your "top" directory before
                    each trial and diff'ing the resulting top files
                    to see this difference. There are three ways to
                    handle this:

                    <table>
                      <tbody>
                        <tr>
                          <td valign="top"><font color=
                          "red"><b>(a)</b></font></td>

                          <td>Include all nucleic acid names in
                          aminoacids.dat and correct the [ atoms ]
                          section of the resulting .top file by
                          hand/script after running pdb2gmx (both
                          the atom type and the charge).</td>
                        </tr>

                        <tr>
                          <td valign="top"><font color=
                          "red"><b>(b)</b></font></td>

                          <td>Remove nucleotide names from
                          aminoacids.dat before importing your pdb
                          file and add them back afterward to use
                          standard GROMACS tools.</td>
                        </tr>

                        <tr>
                          <td valign="top"><font color=
                          "red"><b>(c)</b></font></td>

                          <td>Do not include nucleic acid names in
                          aminoacids.dat and make an index file
                          which specifies the DNA/RNA atoms to use
                          standard GROMACS tools.</td>
                        </tr>

                        <tr>
                          <td></td>
                        </tr>

                        <tr>
                          <td></td>
                        </tr>
                      </tbody>
                    </table>Note that if you choose options
                    <b>(b)</b> or <b>(c)</b> above, you may need to
                    modify the atom names in your .pdb file to
                    agree with the residue listings in the .rtp
                    file due to issue (3) above.
                  </div>
                </td>
              </tr>

              <tr>
                <td valign="top">(7)</td>

                <td>
                  <div align="justify">
                    <b><font color="red">(!) Using AMBER
                    ions:</font></b> Our AMBER ports include common
                    ion definitions, which are listed in the
                    ffamber*.rtp files (just below the TIP water
                    models). This allows the AMBER ports to be used
                    without modification or use of the GROMACS
                    ions.itp file. At the moment these include
                    Cl<small><sup>-</sup></small> ,
                    IB<small><sup>+</sup></small>,
                    Na<small><sup>+</sup></small>,
                    K<small><sup>+</sup></small> ,
                    Rb<small><sup>+</sup></small>,
                    Cs<small><sup>+</sup></small>,
                    Li<small><sup>+</sup></small> ,
                    Ca<small><sup>2+</sup></small>,
                    Mg<small><sup>2+</sup></small>,
                    Zn<small><sup>2+</sup></small>,
                    Sr<small><sup>2+</sup></small>, and
                    Ba<small><sup>2+</sup></small>. If your pdb
                    file has ions present and pdb2gmx does not
                    properly convert those ions, please check the
                    atom and residue name of your ions and rename
                    them if necessary to agree with the .rtp file.
                    If you are using ion-related GROMACS tools,
                    such as genion, you will need to enter the
                    AMBER ion definition to the ions.itp file in
                    the "top" directory of the GROMACS
                    distribution.
                  </div>
                </td>
              </tr>

              <tr>
                <td></td>
              </tr>
            </tbody>
          </table>

          <p align="left"><br /></p>
          <hr />
          <br />

          <div style="text-align: left;">
            <b><big><a name="contact" id="contact">Contact
            Us:</a></big></b><br />
          </div>

          <div align="justify">
            <br />
            Being familiar with the AMBER family of force fields is
            not necessary for the use of our ports, but it is
            highly recommended. More importantly, we expect that
            those using these ports will be very experienced in
            using both GROMACS and UNIX/LINUX. Please do not
            contact us with GROMACS/UNIX/LINUX questions.<br />
            <br />
            We have done our best to provide very complete
            documentation above to help GROMACS users with the
            installation and use of our AMBER ports. Please do not
            email with implementation questions unless you are
            confident that your problem/question has not yet been
            addressed above or in the associated <a href=
            "http://chemistry.csulb.edu/ffamber/FAQ.html">FAQ.</a><br />

            <br />
            We would greatly appreciate any scripts that you can
            contribute for others to use in conjunction with our
            AMBER ports. Please contribute such tools via email
            with a paragraph documenting their use and we will post
            them here. Thank you.<br />
            <br />
            <a href=
            "mailto:esorin@csulb.edu?subject=GROMACS%20AMBER%20port%20bug?">
            Report AMBER port problems/bugs</a><br />
            <a href=
            "mailto:esorin@csulb.edu?subject=GROMACS%20AMBER%20port%20addition">
            Report/request AMBER port additions</a><br />
            <a href=
            "mailto:esorin@csulb.edu?subject=GROMACS%20AMBER%20tool%20submission">
            Submit AMBER port tools/scripts</a><br />
            Other general communications can be directed to
            <a href="mailto:esorin@csulb.edu?subject=GROMACS%20AMBER%20communication">
            Eric J. Sorin</a>.
          </div>
        </td>

        <td></td>

        <td></td>

        <td></td>
      </tr>
    </tbody>
  </table>

  <table class="footer" summary="Footer Layout Table" border="0"
  cellpadding="2" cellspacing="0" width="100%">
    <tbody>
      <tr>
        <td align="left" width="30%"><a href=
        "http://chemistry.csulb.edu/index.php?module=pagemaster&amp;PAGE_user_op=view_page&amp;PAGE_id=61">
        <img alt=
        "Chemistry and Biochemistry, 1250 Bellflower Blvd., Long Beach, CA 90840-3903"
        src="address-blk.gif" border="0" /></a></td>

        <td align="middle" width="40%"><a href=
        "http://www.csulb.edu/"><img alt="Go Beach!" src=
        "gobeach.gif" border="0" /></a></td>

        <td align="right" width="30%"><a href=
        "http://chemistry.csulb.edu/index.php?module=phatform&amp;PHAT_MAN_OP=view&amp;PHPWS_MAN_ITEMS%5B%5D=5"
        target="_blank"><img alt="webmaster@chemistry.csulb.edu"
        src="feedback-blk.gif" border="0" /></a></td>
      </tr>
    </tbody>
  </table>
</body>
</html>
